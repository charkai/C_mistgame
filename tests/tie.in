place C1
place A1
place D1
place B1
place G1
place E1
place H1
place F1
place K1
place I1
place L1
place J1
place O1
place M1
place P1
place N1
place S1
place Q1
place A2
place R1
place B2
place C2
place E2
place D2
place F2
place G2
place I2
place H2
place J2
place K2
place M2
place L2
place N2
place O2
place Q2
place P2
place R2
place S2
place A3
place C3
place B3
place D3
place E3
place G3
place F3
place H3
place I3
place K3
place J3
place L3
place M3
place O3
place N3
place P3
place Q3
place S3
place R3
place C4
place A4
place D4
place B4
place G4
place E4
place H4
place F4
place K4
place I4
place L4
place J4
place O4
place M4
place P4
place N4
place S4
place Q4
place C5
place R4
place D5
place A5
place G5
place B5
place H5
place E5
place K5
place F5
place L5
place I5
place O5
place J5
place P5
place M5
place S5
place N5
place A6
place Q5
place B6
place R5
place E6
place C6
place F6
place D6
place I6
place G6
place J6
place H6
place M6
place K6
place N6
place L6
place Q6
place O6
place R6
place P6
place A7
place S6
place B7
place C7
place E7
place D7
place F7
place G7
place I7
place H7
place J7
place K7
place M7
place L7
place N7
place O7
place Q7
place P7
place R7
place S7
place A8
place C8
place B8
place D8
place E8
place G8
place F8
place H8
place I8
place K8
place J8
place L8
place M8
place O8
place N8
place P8
place Q8
place S8
place R8
place C9
place A9
place D9
place B9
place G9
place E9
place H9
place F9
place K9
place I9
place L9
place J9
place O9
place M9
place P9
place N9
place S9
place Q9
place A10
place R9
place B10
place C10
place E10
place D10
place F10
place G10
place I10
place H10
place J10
place K10
place M10
place L10
place N10
place O10
place Q10
place P10
place R10
place S10
place A11
place C11
place B11
place D11
place E11
place G11
place F11
place H11
place I11
place K11
place J11
place L11
place M11
place O11
place N11
place P11
place Q11
place S11
place R11
place C12
place A12
place D12
place B12
place G12
place E12
place H12
place F12
place K12
place I12
place L12
place J12
place O12
place M12
place P12
place N12
place S12
place Q12
place C13
place R12
place D13
place A13
place G13
place B13
place H13
place E13
place K13
place F13
place L13
place I13
place O13
place J13
place P13
place M13
place S13
place N13
place A14
place Q13
place B14
place R13
place E14
place C14
place F14
place D14
place I14
place G14
place J14
place H14
place M14
place K14
place N14
place L14
place Q14
place O14
place R14
place P14
place A15
place S14
place B15
place C15
place E15
place D15
place F15
place G15
place I15
place H15
place J15
place K15
place M15
place L15
place N15
place O15
place Q15
place P15
place R15
place S15
place A16
place C16
place B16
place D16
place E16
place G16
place F16
place H16
place I16
place K16
place J16
place L16
place M16
place O16
place N16
place P16
place Q16
place S16
place R16
place C17
place A17
place D17
place B17
place G17
place E17
place H17
place F17
place K17
place I17
place L17
place J17
place O17
place M17
place P17
place N17
place S17
place Q17
place A18
place R17
place B18
place C18
place E18
place D18
place F18
place G18
place I18
place H18
place J18
place K18
place M18
place L18
place N18
place O18
place Q18
place P18
place R18
place S18
place A19
place C19
place B19
place D19
place E19
place G19
place F19
place H19
place I19
place K19
place J19
place L19
place M19
place O19
place N19
place P19
place Q19
place S19
place R19
