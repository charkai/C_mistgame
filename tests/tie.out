Wow, a tie!
C1A1D1B1G1E1H1F1K1I1L1J1O1M1P1N1S1Q1A2R1B2C2E2D2F2G2I2H2J2K2M2L2N2O2Q2P2R2S2A3C3B3D3E3G3F3H3I3K3J3L3M3O3N3P3Q3S3R3C4A4D4B4G4E4H4F4K4I4L4J4O4M4P4N4S4Q4C5R4D5A5G5B5H5E5K5F5L5I5O5J5P5M5S5N5A6Q5B6R5E6C6F6D6I6G6J6H6M6K6N6L6Q6O6R6P6A7S6B7C7E7D7F7G7I7H7J7K7M7L7N7O7Q7P7R7S7A8C8B8D8E8G8F8H8I8K8J8L8M8O8N8P8Q8S8R8C9A9D9B9G9E9H9F9K9I9L9J9O9M9P9N9S9Q9A10R9B10C10E10D10F10G10I10H10J10K10M10L10N10O10Q10P10R10S10A11C11B11D11E11G11F11H11I11K11J11L11M11O11N11P11Q11S11R11C12A12D12B12G12E12H12F12K12I12L12J12O12M12P12N12S12Q12C13R12D13A13G13B13H13E13K13F13L13I13O13J13P13M13S13N13A14Q13B14R13E14C14F14D14I14G14J14H14M14K14N14L14Q14O14R14P14A15S14B15C15E15D15F15G15I15H15J15K15M15L15N15O15Q15P15R15S15A16C16B16D16E16G16F16H16I16K16J16L16M16O16N16P16Q16S16R16C17A17D17B17G17E17H17F17K17I17L17J17O17M17P17N17S17Q17A18R17B18C18E18D18F18G18I18H18J18K18M18L18N18O18Q18P18R18S18A19C19B19D19E19G19F19H19I19K19J19L19M19O19N19P19Q19S19R19
Thank you for playing!
